### DISCUSSION TITLE
I'm trying to deploy my nextjs project on cpanel but I got blank page

### DISCUSSION THREAD

**Discussion by sarowarhosen03** [vercel/next.js#54725] [Help]
### Summary

I'm trying to deploy my nextjs project on cpanel but I got blank page . In my local machine it working good 
Here is my GitHub repo 
https://github.com/sarowarhosen03/Nextjs-issu

### Additional information

```js
Please anyone help me
```

**Comment by Wiseard** [comment]
I have the same problem ! Please let me know if you got the answer !

**Reply by sarowarhosen03** [reply]
I got this reply from nextjs discord community you can try https://discord.com/channels/752553802359505017/1146341813200879646

**Comment by WeltonWang02** [comment]
@Wiseard @sarowarhosen03 have seen the same issue recently - try downgrading Next to 13.1.2

**Reply by sarowarhosen03** [reply]
I got this reply from nextjs discord community you can try https://discord.com/channels/752553802359505017/1146341813200879646
